doc/liquid/drops.md

```diff
@@ -2212,7 +2212,7 @@ Returns plan of pricing rule.
 ### name
 Returns name of your organization. That can be changed via the [admin dashboard][provider-account-edit].
 ```liquid
-<div>Domain {{ provider.domain }}</div>
+<div>Domain {{ provider.external_domain }}</div>
```

Comment:
See another comment, on `lib/developer_portal/lib/liquid/drops/provider.rb`, I guess this is generated from there.